staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -1304,6 +1313,14 @@ func (c *Cacher) waitUntilWatchCacheFreshAndForceAllEvents(ctx context.Context,
 	return false, nil
 }
 
+func (c *Cacher) resourceVersion() uint64 {
+	c.RLock()
```

Comment:
Do we need to lock at the Cache level if we only need to read from the `watchCache`? From my understanding the `watchCache` field won't get replaced after initialized.

Reply:
Removed